website/blog/2018-08-27-7.0.0.md

```diff
@@ -0,0 +1,444 @@
+
+## The Wait is Over 🎉
+
+Software is never going to be perfect but we're ready to ship something that's already been used in production for some time now! [`@babel/core`](https://www.npmjs.com/package/@babel/core) is already at 5.1 mil downloads/month because of it's usage in tools like [Next.js 6](https://zeit.co/blog/next6), [vue-cli 3.0](https://medium.com/the-vue-point/vue-cli-3-0-is-here-c42bebe28fbb), [React Native 0.56](https://facebook.github.io/react-native/blog/2018/07/04/releasing-react-native-056), and even [Wordpress's frontend](https://github.com/Automattic/wp-calypso) 🙂!
+
+## Babel's Role
+
+I'd like to start this post by re-introducing Babel's [role](https://www.youtube.com/watch?v=fntd0sPMOtQ) in the JavaScript ecosystem over the last few years.
+
+The initial issue was that unlike server languages, there was no way to guarantee that every user has the same support for JavaScript because users could be using different browsers with varying levels of support (especially older versions of Internet Explorer). If developers wanted to use new syntax (e.g. `class A {}`), users on old browsers would just get a blank screen due to the `SyntaxError`.
+
+Babel provided a way for developers to use the latest JavaScript syntax while allowing them to not worry about how to make it backwards compatible for their users by translating it (`class A {}` to `var A = function A() {}`). 
+
+Because of its ability to transform JavaScript code, it also can be used to implement new features: thus it has become a bridge to help [TC39](https://github.com/tc39) (the committee that specifies the JavaScript language) get feedback on proposed JavaScript ideas and for the community to have a say in the future of the language.
+
+Babel has been pretty ubiquitous in JavaScript development over the last several years. We currently have over [1.3 million dependent repos on GitHub](https://github.com/babel/babel/network/dependents), [17 million downloads on npm per month](https://www.npmjs.com/package/babel-core), and [hundreds of users](http://babeljs.io/users) including many major frameworks (React, Vue, Ember, Polymer), and companies (Facebook, Netflix, Airbnb). It has become such a foundation for JavaScript development that many people don't even know it's being used. Even if you aren't using it yourself, it's highly likely your dependencies are using Babel.
+
+## Maintainers are People
+
+Babel has a huge influence on not just the future of the language itself but its community and ecosystem as well. But even with all of this responsibility, Babel is just a community driven project by a couple of volunteers.
+
+It was only just this past year some of the team were able to meet thanks to various circumstances! 
```

Comment:
It was only just this past year some of the team were able to meet for the first time in person